Userland/Libraries/LibThreading/BackgroundAction.h

```diff
@@ -98,8 +98,12 @@ class BackgroundAction final : public Core::Object
                     error = result.release_error();
 
                 m_promise->cancel(Error::from_errno(ECANCELED));
-                if (m_on_error)
-                    m_on_error(move(error));
+                if (m_on_error) {
+                    origin_event_loop->deferred_invoke([this, error = move(error)] {
+                        m_on_error(Error::copy(error));
```

Comment:
Why can we no longer `move()` the original error?

Reply:
I don't know :shrug:, we discussed it a bit on Discord, I pinged you